src/main/java/io/pravega/connectors/flink/table/catalog/pravega/PravegaCatalog.java

```diff
@@ -108,12 +109,12 @@ public PravegaCatalog(String catalogName, String defaultDatabase, String control
         if (failOnMissingField != null) {
```

Comment:
We can put the json related option copying into one function named `propagateJsonOptions`. Please raise an issue and PR to master first to fix this.